EIPS/eip-747.md

```diff
@@ -36,23 +38,31 @@ interface WatchAssetParameters {
   type: string; // The asset's interface, e.g. 'ERC20'
   options: {
     address: string; // The hexadecimal Ethereum address of the token contract
-    symbol?: string; // A ticker symbol or shorthand, up to 5 alphanumerical characters
-    decimals?: number; // The number of asset decimals
+    chainId?: number; // The chain ID of the asset. If empty, defaults to the current chain ID.
+    name?: sring; // A human-readable name. If the contract has a `name()` method, it MUST override this parameter.
+    symbol?: string; // A ticker symbol or shorthand, up to 5 alphanumerical characters. If the contract has a `symbol()` method, it MUST override this parameter.
+    decimals?: number; // The number of asset decimals. If the contract has a `decimals()` method, it MUST override this parameter.
     image?: string; // A string url of the token logo
```

Comment:
It feels like this should also have a comment like:
> If the contract has an `imageUri()` method, it **MUST** override this parameter.

Is there a standard for attaching images to tokens, something that extends either ERC-20 or ERC-1193?